libs/designsystem/menu/src/menu.component.ts

```diff
@@ -34,9 +38,12 @@ import { EventListenerDisposeFn } from '@kirbydesign/designsystem/types';
   styleUrls: ['./menu.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class MenuComponent implements AfterViewInit, OnDestroy {
+export class MenuComponent implements AfterViewInit, AfterContentInit, OnDestroy {
+  readonly MENU_ID: string = 'MENU_ID';
```

Comment:
Any specific reason for capital casing? Otherwise consider our regular camel casing used everywhere else:
```suggestion
  readonly menuId: string = 'MENU_ID';
```